main/webapp/modules/core/scripts/views/data-table/add-column-by-fetching-urls-dialog.html

```diff
@@ -21,6 +21,11 @@
 	     <input type="checkbox" name="dialog-cache-responses" id="$add-column-cache-responses" checked="checked" />
 		<label for="$add-column-cache-responses" bind="or_views_cacheResponses"></label></td>
         </tr>
+        <tr><td colspan="4"><h3 class="inline-header"><span bind="or_views_httpHeaders"></span></h3>
```

Comment:
This &lt;h3&gt; renders as bold text for me. I think this should be normal text just like the other options, so that "Formulate the URLs to fetch:" remains the single bold label. Otherwise I feel like I am prompted to set custom headers before defining the URLs.

Reply:
Agreed - will take this out